### PR TITLE
Expand FAQ#14 and 20, cosmetic text

### PR DESCRIPTION
**Pull request by licaon-kter**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ Blocking access to the internet can help:
 NetGuard is possibly the first free and open source no-root firewall for Android.
 
 Features:
+--------------------------------
```
Comment (M66B) [inline README.md]: Please remove these, the contents are too small to justify a separate section/link

**Comment by licaon-kter** [comment]
all?

**Comment by M66B** [comment]
All, please.

**Comment by licaon-kter** [comment]
Better?

File: README.md (hunk 1 above)
Comment (M66B) [inline README.md]: This one needs to go too.

File: README.md [hunk 2]
```diff
@@ -46,6 +47,7 @@ Downloads:
 
 Usage:
 
+
 * Enable the firewall using the switch in the action bar
```
Comment (M66B) [inline README.md]: No need for extra white space

**Comment by licaon-kter** [comment]
Missed it.

**Pull request merged**